doc/en/components/scheduling/date-range-picker.md

```diff
@@ -0,0 +1,418 @@
+
+|Keys|Description|
+|----|-----------|
+| <kbd>&uarr;</kbd> / <kbd>&darr;</kbd> / <kbd>&larr;</kbd> / <kbd>&rarr;</kbd>| Navigates through the days in the month |
+| <kbd>ENTER</kbd> | Selects the currently focused day |
+| <kbd>PageUp</kbd> | Moves to the previous month's view |
+| <kbd>PageDown</kbd> | Moves to the next month's view |
+| <kbd>SHIFT + PageUp</kbd> | Moves to the previous year |
+| <kbd>SHIFT + PageDown</kbd> | Moves to the next year |
+| <kbd>Home</kbd> | Focuses the first day of the current month that is in view (or earliest month when more than one month view is displayed) |
+| <kbd>End</kbd> | Focuses the last day of the current month that is in view. (or latest month when more than one month view is displayed) |
+| <kbd>Escape</kbd> | Closes the calender pop-up |
+
+## Layout
+
+### Label
+
+You can define a label for the `DateRangePicker` component using the `Label` property when it is in single input mode. In two inputs mode, you can use the `LabelStart` and `LabelEnd` properties to define labels for the start and end date input fields, respectively.
+
+```html
+<igc-date-range-picker label="Date Range">
+</igc-date-range-picker>
+```
+
+```html
+<igc-date-range-picker use-two-inputs="true" label-start="Start Date" label-end="End Date">
+</igc-date-range-picker>
+```
+
+### Format
+
+You also have the option to customize the date format displayed in the input fields. There are three properties available for this purpose: `Locale`, `InputFormat`, and `DisplayFormat`.
+
+The `Locale` property allows you to set the desired locale identifier, which determines how the date is formatted based on regional conventions.
+For example, to display the date in a Japanese format, you can set the locale property like this:
+```html
+<igc-date-range-picker locale="ja-JP">
+</igc-date-range-picker>
+```
+
+If you want to manually define the date format, you can use the `InputFormat` property by passing a custom format string:
+
+```html
+<igc-date-range-picker input-format="dd/MM/yy">
+</igc-date-range-picker>
+```
+
+The `DisplayFormat` property also accepts a custom format string, but it only applies when the input field is idle (i.e., not focused). When the field is focused, the format reverts to the default or to the one defined by `InputFormat`, if both properties are used together:
+
+```html
+<igc-date-range-picker input-format="dd/MM/yy" date-format="yy/MM/dd">
+</igc-date-range-picker>
+```
+
+### Calendar Layout and Formatting
+
+You can further customize the pop-up calendar using various properties:
+
+* The `Orientation` property allows you to set whether the calendar should be displayed vertically or horizontally.
+
+* The `VisibleMonths` property controls how many months are visible at a time, with a value of either 1 or 2.
+
+* The `ShowWeekNumbers` property enables or disables the week number column in the calendar.
+
```

Comment:
Hm, this list is given here but is not exhaustive. Wouldn't it be better to add all calendar properties here as table, as in the current Properties section.
This would make the latter shorter, as it currently rather looks rather like the API pages.